CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+- Proc names should be written in CamelCase.
+
+- Constants and preprocessor macros/defines should be written in UPPERCASE.
```

Comment:
Just confirming we are talking about define constants here and not constant variables? i.e. var/const/foo